examples/mutate/abstract functionality blueprint/mutate.py

```diff
@@ -35,7 +35,7 @@ def _filter(some_data: pd.DataFrame) -> pd.DataFrame:
 # data 2
 # this is for value
 @mutate(data_2, missing_row=value(["c", 145]))
-def _add_missing_value(some_data: pd.DataFrame, missing_row: List[Any]) -> pd.DataFrame:
```

Comment:
I think this is okay to not have `_` from a API UX consistency view, only because we decorate it with a Hamilton thing -- so you will know that this turns into part of DAG for a node.